src/blueprints/tv2/timeline-object-factories/tv2-viz-timeline-object-factory.ts

```diff
@@ -11,31 +12,31 @@ import {
 } from '../../timeline-state-resolver-types/viz-types'
 import { Tv2GraphicsLayer } from '../value-objects/tv2-layers'
 import { DeviceType } from '../../../model/enums/device-type'
-import { TimelineObject } from '../../../model/entities/timeline-object'
 import {
   Tv2FullscreenGraphicsManifestData,
   Tv2OverlayGraphicsManifestData
 } from '../value-objects/tv2-action-manifest-data'
+import { Tv2GraphicsCommandTimelineObjectFactory } from './interfaces/tv2-graphics-command-timeline-object-factory'
 
-
-enum ChannelName {
+enum Engine {
   OVERLAY = 'OVL1',
   FULLSCREEN = 'FULL1',
   WALL = 'WALL1'
 }
 
-export class Tv2VizTimelineObjectFactory implements Tv2GraphicsTimelineObjectFactory {
-  public createThemeOutTimelineObject(blueprintConfiguration: Tv2BlueprintConfiguration, duration: number): VizMseElementInternalTimelineObject {
+export class Tv2VizTimelineObjectFactory implements Tv2GraphicsCommandTimelineObjectFactory, Tv2GraphicsElementTimelineObjectFactory {
+
+  public createThemeOutTimelineObject(blueprintConfiguration: Tv2BlueprintConfiguration): VizMseElementInternalTimelineObject {
     return {
       id: '',
       enable: {
         start: 0,
-        duration
+        duration: 3000
```

Comment:
Magic literal. Same goes for the other durations.

Reply:
I'm not sure what is the best way to handle it in the timelineObject generation, because they basically are just creating objects with a lot of specific values. We could spam the file with variables if we extracted every single one.

Any suggestions?

Reply:
There are a couple of `3000` literals and a couple of `1000` literals in this file. Do they have the duration for the same reason? And if one were to be changed, should the others be changed? If so, then I think we can make som categoric constants. If they all have unique reasons for changing the matching values, then I have no issue with keeping the magic literals.